app/src/androidTest/java/ch/sdp/vibester/activity/LyricsBelongGameActivityTest.kt

```diff
@@ -159,27 +159,27 @@ class LyricsBelongGameActivityTest {
         onView(withId(R.id.progressBarLyrics)).check(matches(isDisplayed()))
     }*/
 
-    @Test
-    fun handleLyricsNoFoundCorrectly() {
-        createMockInvocation()
-        val gameManager = setGameManager()
-        val intent = Intent(
-            ApplicationProvider.getApplicationContext(),
-            LyricsBelongGameActivity::class.java
-        )
-        val scn: ActivityScenario<LyricsBelongGameActivity> = ActivityScenario.launch(intent)
-        val ctx = ApplicationProvider.getApplicationContext() as Context
-        scn.onActivity { activity ->
-            activity.testGetAndCheckLyrics(ctx, "the best song in the world", "Mr.Mystery", "", gameManager)
-        }
-        /** FIXME: API takes a lot of time to process this request
-        comment the following lines if this test fail */
-    //   Thread.sleep(sleepTime)
-    //    onView(withId(R.id.nextSongButton)).check(matches(isDisplayed()))
-        //song skipped, not consider as wrong
-    //    assertEquals(true, gameManager.getScore() == 0)
-    //    assertEquals(true, gameManager.getWrongSongs().size == 0)
-    }
+//    @Test
```

Comment:
consider replacing the // before each line with a block comment /* */, it will be easier to modify later.

Reply:
fixed!